apps/explorer/src/modules/bridge/hooks/useCrossChainTokens.ts

```
@@ -0,0 +1,27 @@
+  destinationToken: T
+}
+
+export function useCrossChainTokens(crossChainOrder: CrossChainOrder): CrossChainTokens {
+  const {
+    bridgingParams: { sourceChainId, destinationChainId, inputTokenAddress, outputTokenAddress },
+    order,
+  } = crossChainOrder
+  const { data: sourceTokens } = useTokenList(sourceChainId)
+  const { data: destinationTokens } = useTokenList(destinationChainId)
+
+  const sourceToken = sourceTokens && sourceTokens[inputTokenAddress.toLowerCase()]
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)